app/styles/overview.less

```diff
@@ -1,3 +1,7 @@
+@radiobutton-bg-lightblue:  #55d4f5;
```

Comment:
we define the variable in "light-theme-variables.less" and "dark-theme-variables.less"